stable/prometheus-operator/templates/exporters/kubelet/servicemonitor.yaml

```
@@ -6,7 +6,7 @@ metadata:
   namespace: {{ template "prometheus-operator.namespace" . }}
   labels:
     app: {{ template "prometheus-operator.name" . }}-kubelet
-{{ include "prometheus-operator.labels" . | indent 4 }}
+{{- include "prometheus-operator.labels" . | indent 4 }}
```

I believe you need the newline after `-kubelet` above, but this `{{-` whitespace chomper will remove it.

```suggestion
{{ include "prometheus-operator.labels" . | indent 4 }}
```

---

@tklovett I get this output with the newline chomp:

    $ helm template prometheus-operator --namespace monitoring ./stable/prometheus-operator
```yaml
# Source: prometheus-operator/templates/exporters/kubelet/servicemonitor.yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: prometheus-operator-kubelet
  namespace: monitoring
  labels:
    app: prometheus-operator-kubelet    
    chart: prometheus-operator-8.15.7
    release: "prometheus-operator"
    heritage: "Helm"
spec:
  endpoints:
```

And this without:
```yaml
# Source: prometheus-operator/templates/exporters/kubelet/servicemonitor.yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: prometheus-operator-kubelet
  namespace: monitoring
  labels:
    app: prometheus-operator-kubelet
    
    chart: prometheus-operator-8.15.7
    release: "prometheus-operator"
    heritage: "Helm"
spec:
  endpoints:
```

Not sure why you think it's necessary?

---

Ah, my bad. I didn't realize there was also a newline coming in from the [`prometheus-operator.labels`](https://github.com/helm/charts/blob/05966cd7f3e712ce82aec1c884d913b9c62ed704/stable/prometheus-operator/templates/_helpers.tpl#L48-L49) template too. You have it right 👍

---

Cool, marking as resolved.